src/main/java/duke/task/Parseable.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * @author leslieharland
+ */
+
+package duke.task;
+
+/**
+ * The Interface Parseable.
+ */
+public interface Parseable {
+
+    /**
+     * Parses the.
```

Comment:
It would be better if the java comment could explain what the method is parsing.